sdk/iot/provisioning/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure IoT Clients for Embedded C: Azure IoT Device Provisioning Client
+
+Official Embedded C client for Azure IoT DPS - Device Provisioning Services.
+
+For more details about DPS check [Azure documentation](https://docs.microsoft.com/azure/iot-dps/)
+
+## Getting started
+
+- Pelase refer to the state machine diagram which describes the Azure IoT Embedded C SDK flow for DPS [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service). 
+
+- **Samples**: [This page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/provisioning/samples) explains samples for the Azure IoT Embedded C SDK IoT Provisioning Client and how to use them.
+
+## Need help?
```

Comment:
Same as with others: this should only be in the top-level readme.md (root of the repo)